thunder/images/images.py

```diff
@@ -79,35 +80,44 @@ def toblocks(self, size='150', padding=None):
         from thunder.blocks.blocks import Blocks
         from thunder.blocks.local import LocalChunks
 
+        if chunk_size is 'auto':
+            chunk_size = str(int(1000000.0/self.shape[0]))
```

Comment:
Nevermind about `dtype`, forgot that was handled inside `bolt`. Would still be good to see the real use case confirmation.